satpy/_config.py

```diff
@@ -18,6 +18,11 @@
 """Satpy Configuration directory and file handling."""
 from __future__ import annotations
 
+try:
+    from functools import cache  # type: ignore
+except ImportError:  # for python < 3.9
+    from functools import lru_cache as cache
```

Comment:
You could import this from the compat module:

https://github.com/ameraner/satpy/blob/d85fc1843aeb2fa2f8cf4dd5d395ffa7580e5083/satpy/_compat.py#L93-L96

Reply:
ah, yes, of course! thank you